lib/osf-components/addon/components/sort-button/template.hbs

```diff
@@ -1,18 +1,16 @@
 <Button
-    data-test-ascending-sort={{this.sortBy}}
-    data-analytics-name={{concat 'Ascending sort ' this.sortBy}}
-    local-class={{if (eq @sort this.sortBy) 'selected' 'not-selected'}}
-    title={{t 'general.sort_asc'}}
+    data-test-sort-button={{this.sortBy}}
```

Comment:
Product told me they were discussing making it standard: https://centerforopenscience.slack.com/archives/C07C771CFK4/p1724691936956749, so it might be acceptable, they didn't 100% confirm that it should be everywhere though.